packages/runtime/runtime-utils/src/dataStoreHelpers.ts

```diff
@@ -61,6 +62,12 @@ export function responseToException(response: IResponse, request: IRequest): Err
     return responseErr;
 }
 
+export function packagePathToTelemetryProperty(
```

Comment:
nit: Comment. Explaining format of packagePath (well, it's pretty obvious) and mention that it tags it as "CodeArtifact"